src/pretix/base/orderimport.py

```diff
@@ -805,7 +805,7 @@ def clean(self, value, previous_values):
                 return self.q.clean_answer(value)
 
     def assign(self, value, order, position, invoice_address, **kwargs):
-        if value:
+        if value is not None:
```

Comment:
This is the type of change where an accompanying unit test would not only be easy to write, but also be extremely helpful to prevent this issue from surfacing again if we ever refactor the order import :)